dali/operators/displacement/displacement_test.cc

```diff
@@ -43,12 +43,20 @@ TYPED_TEST(DisplacementTest, Jitter) {
 */
 
 TYPED_TEST(DisplacementTest, WarpAffine) {
+  const OpArg params[] = {
+    {"matrix", "1.0, 0.8, 0.0, 0.0, 1.2, 0.0", DALI_FLOAT_VEC},
+    { "fill_value", "0", DALI_FLOAT }
+  };
+  this->RunTest("WarpAffine", params, 2, false, 0.1 /* 0.1 percent */);
+}
+
+TYPED_TEST(DisplacementTest, OldWarpAffine) {
   const OpArg params = {"matrix", "1.0, 0.8, 0.0, 0.0, 1.2, 0.0", DALI_FLOAT_VEC};
-  this->RunTest("WarpAffine", &params, 1);
+  this->RunTest("OldWarpAffine", &params, 1, false, 0.1 /* 0.1 percent */);
```

Comment:
After we:
1. verify (with compare_pipelines in python) that both WarpAffine and OldWarpAffine produce the same results (with a reasonable eps) 
2. Fix and run the CPU benchmarks and there is no performance degradation
I'd say we remove `OldWarpAffine` completely